test/distributed/test_dynamo_distributed.py

```diff
@@ -568,6 +569,7 @@ def opt_fn(inputs):
     @patch.object(config, "optimize_ddp", True)
     @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
     def test_graph_split_inductor(self):
+        assert config.optimize_ddp
```

Comment:
is it really necessary to assert this after we patch it above?

Reply:
I had some issue with patch.object on config previously. Maybe I'll convert to `@config.patch(..)` instead since it is also supported.

Reply:
Anw, I think the assert is not a bad thing